### PR TITLE
New tools for statistic handling and debugged plotting results

### PR DESCRIPTION
**Pull request by wattik**
Now, if you wanna hurry up, there's SHOWRESULTS(filename) for you.

### CODE REVIEW

File: data/generatingScripts/evaluateResults.m [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+% vectors.FP/.FN/.TP/.TN vectors of bools showing where those occured
+% statistic.FP/.FN/.TP/.TN float,a percentage of the whole set
+% statistic.tresh_hold 
+
+assumed_anomaly = anomaly >= tresh_hold;
+
+vectors.TP = (assumed_anomaly == 1) & (has_anomaly == 1);
+vectors.TN = (assumed_anomaly == 0) & (has_anomaly == 0);
+vectors.FP = (assumed_anomaly == 1) & (has_anomaly == 0);
+vectors.FN = (assumed_anomaly == 0) & (has_anomaly == 1);
```
Comment (breznak) [inline data/generatingScripts/evaluateResults.m:17]: This is a nice functionality! :+1: 
Maybe we can compute precision/recall/F right away here too?

File: opf/modelParamsRDSE.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
                     'fieldname': u'function',
                     'n': 1000,
                     'name': u'function',
-                    'resolution': 0.005,
+                    'resolution': 0.05,
```
Comment (breznak) [inline opf/modelParamsRDSE.py:12]: @wattik this is the value we've computed on the paper? :)

**Comment by breznak** [comment]
@wattik nice work on the "resulting", please add it to all the sections in MAIN.m to print the results and maybe also save a file "results/<experimentname>_results.txt"
You may consider also computing the precision/recall/F during that "results" function.

File: data/generatingScripts/evaluateResults.m (hunk 1 above)
Comment (wattik) [inline data/generatingScripts/evaluateResults.m:17]: What is F?

File: opf/modelParamsRDSE.py (hunk 2 above)
Comment (wattik) [inline opf/modelParamsRDSE.py:12]: Yes. Pure rocket science.

**Comment by wattik** [comment]
Well, why would we add results to MAIN.m? I think we first have to run the script, then run python scripts and then evaluate results. No need for it in MAIN.m. In addition to that, as we have agreed I'm going to put the statistics in figures instead.

**Comment by breznak** [comment]
>  In addition to that, as we have agreed I'm going to put the statistics in figures instead.
> :+1: ok, that'd be even better

**Pull request merged**